### ISSUE TITLE
[Bug] Could not find a 3d context

### ISSUE THREAD

**Issue by Tak1za** [visgl/react-google-maps#264]
### Description

Sometimes when loading the map, we encounter this error:
![image](https://github.com/visgl/react-google-maps/assets/18700114/fa1cbf4e-5605-4e1c-b3e5-196241984992)

I did some googling around and found that it could be because of the browser's hardware acceleration being turned off.

I did turn it on, restarted my browser but still encountered this same error sometimes.

I'm not sure if this dependency with 3d context is for supporting deck.gl, but if it is, is there a way I can exclude that so I don't get this error anymore?

### Steps to Reproduce

Try rendering a map view with some markers.

### Environment

- Library version: "@vis.gl/react-google-maps": "^0.5.4"
- Google maps version: weekly
- Browser and Version: Chrome: 122.0.6261.128 (Official Build) (64-bit)
- OS: Windows

**Comment by usefulthink** [comment]
Yeah, that is something in the google maps API that we have no influence on. I think the maps API should have a fallback in this case, but that seems to fail under some conditions. 

The webgl based renderer is used in google maps at least for all vector-based maps (not sure if also the raster maps are using webgl for rendering), but there should be a fallback there.

Question would be, how can we implement a fallback in case this fails. 
If we're lucky this exception is thrown in the context of the map constructor, but from the stack-trace in your screenshot it looks like there are no frames of our library on the stack, so no way to catch that exception (except for maybe hacking something with `window.onerror` or something like that).

**Comment by Tak1za** [comment]
Umm.. I actually investigated a little further, and used their sample app in Cloud Shell they provide which uses the Web GL renderer.
Here's the link for that: https://developers.google.com/maps/documentation/javascript/examples/webgl/webgl-overlay-simple

Use the Google Cloud Shell in the link above to give it a try. Turn the map into a satellite one and if you refresh multiple times, it will throw the same error I have attached but it still loads the map and does not fail the map loading.

Whenever this lib is used though, the map load always fails.

**Comment by downeyfe** [comment]
@Tak1za can you please link me to the issue on Google's side? I'm encountering the same issue when switching to satellite if graphics acceleration is disabled.

**Comment by Tak1za** [comment]
@downeyfe https://issuetracker.google.com/issues/191028632
Here it is, but they didn't respond unfortunately.

I ended up informing my users that they need to keep 3D acceleration on in their browser and there is a chance for intermittent failures.

**Comment by downeyfe** [comment]
@Tak1za that's frustrating, but thanks for sharing!
